README.md

```diff
@@ -98,63 +120,85 @@ To provide further configuration, add the `custom.kubeapilinter` section to your
 
 [golangci-config-struct]: https://pkg.go.dev/sigs.k8s.io/kube-api-linter/pkg/config#GolangCIConfig
 
-Where fixes are available within a rule, these can be applied automatically with the `--fix` flag.
+Where fixes are available within a rule, these can be applied automatically with the `--fix` flag:
 
 ```shell
-golangci-kube-api-linter run path/to/api/types --fix
+./bin/golangci-lint-kube-api-linter run path/to/api/types --fix
 ```
 
 ### Golangci-lint Plugin
 
 The Kube API Linter can also be used as a plugin for `golangci-lint`.
-To do this, you will need to install the `golangci-lint` binary and then install the Kube API Linter plugin.
+To do this, you will need to install the `golangci-lint` binary and then build the Kube API Linter plugin.
 
 More information about golangci-lint plugins can be found in the [golangci-lint plugin documentation][golangci-lint-plugin-docs].
 
 [golangci-lint-plugin-docs]: https://golangci-lint.run/plugins/go-plugins/
 
+**Important**: The plugin must be built from the vendor directory, not directly from the module path.
+
+**Step 1**: Ensure the module is in your project's vendor directory:
+
+```shell
+go mod vendor
+```
+
+**Step 2**: Build the plugin from the vendor directory:
+
 ```shell
-go build -buildmode=plugin -o bin/kube-api-linter.so sigs.k8s.io/kube-api-linter/pkg/plugin
```

Comment:
Losing `pkg/plugin` here is not correct. The root module is for module mode, the plugin path is for the plugin mode, they are different codepaths to build